File: 2.Modules/function/Contracts/function/csharp/HDS/TST/CSF010/featureMain.cs
using HandStack.Core.ExtensionMethod;
using HandStack.Core.Extensions;
using HandStack.Core.Helpers;
using HandStack.Web.MessageContract.Contract;
using HandStack.Web.MessageContract.DataObject;

using Microsoft.AspNetCore.Mvc;

using Newtonsoft.Json.Linq;

using System.Collections.Generic;
using System.Data;

namespace HDS.Function.TST
{
    public class CSF010
    {
        public DataSet? GF01(List<DynamicParameter> dynamicParameters, DataContext dataContext)
        {
            string typeMember = "TST.CSF010.GF01";

            string serverDate = dynamicParameters.Value("ServerDate").ToStringSafe();
            string serverName = dynamicParameters.Value("ServerName").ToStringSafe();

            DataTableHelper dataTableBuilder = new DataTableHelper();
            dataTableBuilder.AddColumn("FunctionResult", typeof(string));

            dataTableBuilder.NewRow();
            dataTableBuilder.SetValue(0, 0, $"typeMember: {typeMember}, serverDate: {serverDate}, serverName: {serverName}");

            using DataSet result = new DataSet();
            using (DataTable table = dataTableBuilder.GetDataTable())
            {
                result.Tables.Add(table);
            }

            return result;
        }
    }
}

File: 2.Modules/function/Contracts/function/HDS/TST/JSF010/featureMain.js
var fs = require('fs');
var syn = require('syn');

module.exports = {
    GF01: (callback, moduleID, parameters, dataContext) => {
        var typeMember = "TST.JSF010.GF01";
        var serverDate = $array.getValue(parameters, 'ServerDate');
        var serverName = $array.getValue(parameters, 'ServerName');

        var result = {
            DataTable1: [
                {
                    FunctionResult: `typeMember: ${typeMember}, moduleID: ${moduleID}, serverDate: ${serverDate}, serverName: ${serverName}`
                }
            ]
        };

        callback(null, result);
    }
}
